build.gradle

```diff
@@ -180,7 +180,6 @@ dependencies {
 
     // ZipArchive dependencies used for integration tests
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-ml-plugin', version: "${opensearch_build}"
-
```

Comment:
So spotless removed it again? Weird

Reply:
this time while testing I actually added:
```
    zipArchive group: 'org.opensearch.plugin', name:'opensearch-knn', version: "${opensearch_build}"
        zipArchive group: 'org.opensearch.plugin', name:'neural-search', version: "${opensearch_build}"
```
a bunch of times, maybe in the end after I removed it just kept line deleted

Reply:
@amitgalitz can you add a note on top of the template which requires the associated plugins in the OS cluster?

Reply:
do you mean in the use case name?

Reply:
Will add in comments and documentation